server/app/models/grid_service.rb

```diff
@@ -67,8 +67,12 @@ def to_path
   end
 
   # @return [String]
-  def name_with_stack
-    "#{self.stack.name}-#{self.name}"
+  def name_with_stack(ignore_default = false, separator = '-')
```

Comment:
Could these parameters be in a option hash (or named parameters). `grid_service.name_with_stack(true, '/')` is not very readable...